resources/windows/wix/service.pxp-agent.wxs.erb

```diff
@@ -0,0 +1,61 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+
+  <Fragment>
+    <ComponentGroup Id="Service_PXPAGENT">
+      <ComponentRef Id="Service_NSSM" />
```

Comment:
Can we have this be consistent? I know it's isolated to the service itself, but we currently have three different ways we're defining this. `<ComponentRef Id="MCOService" />`, <ComponentRef Id="PuppetServiceAutomatic" /> (and friends), and `<ComponentRef Id="Service_NSSM" />`.